src/components/App/App.tsx

```diff
@@ -1,12 +1,43 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import {BrowserRouter as Router, Route, Switch} from 'react-router-dom';
+import { getArticles, getSentiment } from '../../apiCalls';
+import { CleanedArticle } from '../../Models';
 import Form from '../Form/Form';
 import Feed from '../Feed/Feed';
+import Article from '../Article/Article';
 import './App.css';
 
 const App = (): JSX.Element => {
+  const [articles, setArticles] = useState<CleanedArticle[]>([]);
+  const [error, setError] = useState('');
   const [ userSentiment, setUserSentiment ] = useState<number | null>(null);
```

Comment:
For consistency: yes to [spaces inside of braces](https://github.com/airbnb/javascript#whitespace--in-braces), no to [spaces inside brackets](https://github.com/airbnb/javascript#whitespace--in-brackets).

```suggestion
  const [userSentiment, setUserSentiment] = useState<number | null>(null);
```